### ISSUE TITLE
Joy of Painting: Canvases are blank when joining server

### ISSUE THREAD

**Issue by CreatedorMade** [ercanserteli/xercamods#79]
Minecraft 1.19.2 / Forge 43.2.0
Canvases are all white for everyone who joins the server initially, breaking and replacing them fixes them. This happens with both unsigned and signed canvases. It doesn't happen in singleplayer.

**Comment by CreatedorMade** [comment]
I found the issue. I was using a Python script to import paintings by generating new NBT files. It worked, but it erroneously generated paintings with a 166-character long "name" field. When replicating paintings from the server to the client, if there were any of these bugged paintings being imported, it would disconnect the player, once, then let them join after, but leaving the paintings blank because they failed to load. After fixing my script and having it generate name fields of an appropriate length, paintings now load and we no longer get disconnected trying to join.

I also noticed another bug with these paintings, we would get disconnected trying to Sign and Close them.

I guess this isn't really a bug, but I would recommend sanitizing the .paint files on import ESPECIALLY when uploading to servers.
